content/questions/admin/question-005.md

```diff
@@ -0,0 +1,13 @@
+---
+question: "Which of the following are correct methods to generate and share a diagnostic file for GitHub Enterprise Server?"
+archetype: "questions"
+title: "Question 005"
+draft: false
+---
+
+> https://docs.github.com/en/enterprise-server@3.11/support/contacting-github-support/providing-data-to-github-support#creating-and-sharing-diagnostic-files
+- [x] Create a diagnostic file by running the `ghe-diagnostics` command-line utility, then download it from the Management Console.
```

Comment:
Is this correct? Docs says **You can use this method without signing into the Management Console.**